doc/source/user_guide/groupby.rst

```diff
@@ -1308,17 +1306,8 @@ is only needed over one column (here ``colname``), it may be filtered
             ],
         }
     )
-
-    # Decimal columns can be sum'd explicitly by themselves...
     df_dec.groupby(["id"])[["dec_column"]].sum()
 
-    # ...but cannot be combined with standard data types or they will be excluded
-    df_dec.groupby(["id"])[["int_column", "dec_column"]].sum()
-
-    # Use .agg function to aggregate over standard and "nuisance" data types
-    # at the same time
-    df_dec.groupby(["id"]).agg({"int_column": "sum", "dec_column": "sum"})
```

Comment:
You used to need to do this to include nuisance columns; not anymore. Now they are included by default.